src/nvim/tui/terminfo_defs.h

```diff
@@ -1,8 +1,14 @@
+// This is an open source non-commercial project. Dear PVS-Studio, please check
+// it. PVS-Studio Static Code Analyzer for C, C++ and C#: http://www.viva64.com
+
 // uncrustify:off
 
-// Generated by scripts/update_terminfo.sh and ncurses 6.4.20230520
+//
+// Generated by scripts/update_terminfo.sh and ncurses 6.3.20211021
+//
 
-#pragma once
+#ifndef NVIM_TUI_TERMINFO_DEFS_H
+#define NVIM_TUI_TERMINFO_DEFS_H
```

Comment:
?

Reply:
Sorry seems like I mixed up the master br with the stable one as I could not compile the master one correctly with msvc for some reason hence I revert to stable.

Will fix this